lib/main.coffee

```diff
@@ -0,0 +1,60 @@
+{CompositeDisposable} = require 'atom'
+{parseString} = require 'xml2js'
+path = require 'path'
+
+module.exports =
+  config:
+    executablePath:
+      title: 'CSSLint Executable Path'
+      type: 'string'
+      # default: path.join(__dirname, '..', 'node_modules', '.bin', 'csslint.cmd')
```

Comment:
You could use the `execNode` method of the helpers module and pass in the direct `.js` path to it (__Note__: I am not sure if cmd one will work)